src/Carbon/Lang/so.php

```diff
@@ -10,6 +10,50 @@
  */
 
 /*
- * Unknown default region, use the first alphabetically.
+ * Author:
+ * - Abdifatah Abdilahi(@abdifatahz)
  */
-return require __DIR__.'/so_DJ.php';
+return [
+    'year' => '{1}:count sanad|{0}:count sanadood|]1,Inf[:count sanadood',
+    'a_year' => '{1}a sanad|{0}:count sanadood|]1,Inf[:count sanadood',
```

Comment:
According to what you wrote in #2106 "a year" should be "sanad", so I will remove all the "a" which looks like coming from the English version.

Reply:
Yes that's right.